media/js/firefox/new/scene1.js

```diff
@@ -120,4 +120,19 @@
         initOtherPlatformsModal();
     }
 
+    /**
+     * If user is on iOS and has followed an adjust.com link, redirect to
+     * the App Store.
+     * https://github.com/mozilla/bedrock/issues/5629
+     */
+    // TODO: is this a solid enough check?
```

Comment:
The only other thing I can think to do here would be to also check the referrer came from adjust.com. It does worry me a little that this kind of auto-redirect thing could one day get us penalized in search rank. If someone shared the link or published / tweeted it, could it get picked up and indexed?

Reply:
Hmmm, this is a very valid concern.

@rraue - do you have any thoughts here? For context, someone will land on `/firefox/new/?open-app-store=true` and be automatically redirected to the Apple App store. If this URL gets shared somehow, would/could there be a negative SEO impact?

Reply:
Unfortunately, (according to Aaron at Adjust), we cannot rely on `adjust.com` being present in `document.referrer`. 😞

Reply:
It'd still be nice to try and get @rraue's thoughts here re and potential SEO impact?